gateway-service-api/build.gradle.kts

```diff
@@ -44,7 +44,12 @@ sourceSets {
 }
 
 dependencies {
-  api("io.grpc:grpc-protobuf:1.50.0")
-  api("io.grpc:grpc-stub:1.50.0")
+  api("io.grpc:grpc-protobuf:1.56.0")
```

Comment:
Suggest also adding a grpc bom here. Example: https://github.com/hypertrace/attribute-service/blob/2ac91ac20cd8e33601aaff38e423b5beb305795e/attribute-service-api/build.gradle.kts#L36-L38

With the grpc bom, we can then remove all the places the grpc version is explicitly declared (except in the protoc artifact on line 25). 

Other than deduping, what this does is when an upstream repo consumes this artifact, if they're using an older version of grpc they'll be upgraded - rather than ending up with some old ones, and then some new ones coming from this lib, like grpc-protobuf and grpc-stub. That mismatch will lead to runtime failures. Hope that makes sense.